user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Variables/Variables.md

```diff
@@ -0,0 +1,92 @@
+---
+uid: Variables
+---
+
+# Variables
+
+Variables are reusable data objects that persist across sessions and can be dynamically updated. They are defined independently from components and can be [passed to components as data](xref:Apply_Data_Feed), linked in a [query](xref:Creating_GQI_query), used in a [flow](xref:Using_flows), and more.
+
+Variables are available starting from DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12<!--RN 41039-->.
+
+## Creating a variable
+
+To create a new variable:
+
+1. In edit mode, go to the *Data* pane on the right, expand the *Variables* item, and click the + icon.
+
+1. Specify a name for the variable. This name must be unique within the dashboard or the low-code app.
+
+1. In the dropdown menu, select the **variable type**<!--RN 41063-->:
+
+   - Element
+
+   - Service
+
+   - View
+
+   - Object manager instance
+
+   - Text
+
+   - Number
+
+   - Table <!--RN 41132-->
+
+1. Optionally, enable the ***Read-only* setting** if you want the variable's value to remain fixed and unchangeable when interacting with the app. This setting is only available for Low-Code Apps, as Dashboard variables are always read-only.
+
+1. Depending on the selected variable type, specify the **default value** for the variable.
+
+   > [!NOTE]
+   > In a low-code app, specifying a default value for a variable is optional, unless the variable is set to *Read-only*.
+
+   - Default element: Enter the name of an element in your DataMiner System or select one from the dropdown list.
+
+   - Default service: Enter the name of a service in your DataMiner System or select on from the dropdown list.
+
+   - Default view: Enter the name of a view in your DataMiner System or select from the dropdown list.
+
+   - Default object manager instance:
+
+     1. Enter the name of a module in your DataMiner System or select from the dropdown list.
+
+     1. Enter the name of an instance in your DataMiner System or select from the dropdown list.
+
+   - Default text: Enter custom text. HTML text formatting is supported, except when the variable is passed to a text input or search input component.
```

Comment:
I'm not sure if we need to mention HTML text formatting here. The html format has nothing to do with the text variable. It holds text, and depending where it is used, that text can be interpreted as html.

Reply:
```suggestion
   - Default text: Enter custom text.
```